### ISSUE TITLE
Assignment to const variable

### ISSUE THREAD

**Issue by xaircore** [gemini-testing/gemini#629]
All tests failed with error: Assignment to const variable. [Stacktrace](http://shot.qip.ru/00DuQd-6oZhWgu7b/). On gemini@4.11.3 all fine

Output of `gemini --version`:

4.12.1

Test source code:

```
gemini.suite('SBIS3.CONTROLS.ProgressBar Online', function () {

    gemini.suite('base', function (test) {

        test.setUrl('/regression_progress_bar_online.html').setCaptureElements('.capture')

            .before(function (actions) {
				
				this.pb = '[name="ProgressBar 1"]';
				
                actions.waitForElementToShow(this.pb, 40000);
            })

            .capture('plain')

            .capture('with_progress', function (actions) {
                actions.executeJS(function (window) {
                    window.$ws.single.ControlStorage.getByName('ProgressBar 1').setProgress(50);
                });
            })
			
			.capture('disabled', function (actions) {
                actions.executeJS(function (window) {
                    window.$ws.single.ControlStorage.getByName('ProgressBar 1').setEnabled(false);
                });
            })
    });
});
```

**Comment by sipayRT** [comment]
will be fixed in next version - https://github.com/gemini-testing/gemini/pull/624

**Comment by SevInf** [comment]
Fixed in 4.12.2
